src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,141 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final double DEFAULT_LOAD_FACTOR = 0.75;
+    private static final int ZERO_HASH = 0;
+    private Node<K, V>[] table;
+    private Node<K, V> currentNode;
+    private int threshold;
+    private int capacity;
+    private int size;
```

Comment:
remove threshold, capacity, ZERO_HASH, currentNode; we don't need them as fields/ constants